### ISSUE TITLE
How to set the value of patience

### ISSUE THREAD

**Issue by dsy412610** [fchollet/keras#2116]
I want to stop training when validate loss leveled  off and fluctuate by using EarlyStopping, so how to set the value of patience? According to experience or some rules?

**Comment by carlthome** [comment]
Simply pass "patience = # epochs without improvement before stopping" when constructing EarlyStopping, such as `EarlyStopping(patience=10)`

**Tip:** [check the source comments](https://github.com/fchollet/keras/blob/master/keras/callbacks.py#L302).

**Comment by dsy412610** [comment]
Thanks! I will try it.
